exercises/simple-linked-list/src/example/java/SimpleLinkedList.java

```diff
@@ -0,0 +1,79 @@
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
+/**
+ *
+ * @author matt
```

Comment:
You can teach your IDE not to generate these comments. They probably aren't relevant for this repo.